src/ui/RealmBrowser/Content/Table/HeaderCell.tsx

```diff
@@ -41,17 +43,42 @@ const getPropertyType = (property: Realm.ObjectSchemaProperty) => {
   }
 };
 
+const optionalMarkOptOut = ['list', 'dictionary', 'set'];
+const getOptionalMark = (property: IPropertyWithName) => {
+  if (optionalMarkOptOut.includes(property.type)) {
+    return '';
+  }
+
+  return property.optional ? '?' : '';
+};
+
+const getCollectionShorthand = (property: IPropertyWithName) => {
+  switch (property.type) {
+    case 'list':
+      return '[]';
+
+    case 'dictionary':
+      return '{}';
+
+    case 'set':
+      return '<>';
+
+    default:
+      return '';
+  }
+};
```

Comment:
Just FYI. I hope we can decide to change these before the official release.

Reply:
Are we talking about `string[]` => `List<string>`? If so, that needs to be corrected several places.

Reply:
Yep - at least that's my hope 🤞